### ISSUE TITLE
Unable to retrieve access token

### ISSUE THREAD

**Issue by jordanaroth** [vmware-samples/sddc-import-export-for-vmware-cloud-on-aws#14]
As of today, I've been getting the error when running the export command:

`Unable to retrieve access token. Server response: None`

I've tried this on multiple computers and multiple orgs just to rule out API token issues or connection issues. All the same result. I've also tested that these Access Tokens do work as I'm able to authenticate via other methods (e.g. Terraform) using the same Access Token.

**Comment by jordanaroth** [comment]
I'll add that I event tried passing the source-token in the command line and the same error:

```
python sddc_import_export.py --source-refresh-token <removed> -o export
Loaded source refresh token from command line
Unable to retrieve access token. Server response:None
```

**Comment by kremerpatrick** [comment]
Fixed eeafdd19ab72abc78f19df30cd63a6e11c03cae9
